doc/stake-pool-operations/retire_stakepool.md

```diff
@@ -10,7 +10,7 @@ The deregistration certificate contains the _epoch_ in which we want to retire t
 First, you need to figure out the current epoch. The simplest way to get the current epoch is to use the `tip` command:
 
 ```bash
-> cardano-cli query tip --testnet-magic 42 --mary-era --cardano-mode
+> cardano-cli query tip --testnet-magic 42 --cardano-mode
```

Comment:
I don't think `--cardano-mode` is required any longer.

Reply:
It's strictly not required, the default is cardano mode